### ISSUE TITLE
TS Server fatal error:  Debug Failure.

### ISSUE THREAD

**Issue by ynwd** [denoland/vscode_deno#295]
OS: Latest Ubuntu Linux
VSCODE_deno extension version: Canary

**TS Server Log**
❗️Server logging disabled. To help us fix crashes like this, please enable logging by setting:
```json
"typescript.tsserver.log": "verbose"
```
After enabling this setting, future crash reports will include the server log.

**TS Server Error Stack**
Server: `syntax`
```
Error: Debug Failure.
    at actualResolveModuleNamesWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:105102:161)
    at resolveModuleNamesWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:105340:26)
    at resolveModuleNamesReusingOldState (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:105530:19)
    at processImportedModules (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106941:35)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106713:17)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106911:21
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at processLibReferenceDirectives (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106906:16)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106710:21)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106911:21
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at processLibReferenceDirectives (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106906:16)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106710:21)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106911:21
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at processLibReferenceDirectives (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106906:16)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106710:21)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106911:21
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at processLibReferenceDirectives (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106906:16)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106710:21)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106911:21
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at processLibReferenceDirectives (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106906:16)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106710:21)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106911:21
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at processLibReferenceDirectives (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106906:16)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106710:21)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106911:21
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at processLibReferenceDirectives (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106906:16)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106710:21)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106911:21
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at processLibReferenceDirectives (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106906:16)
    at findSourceFileWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106710:21)
    at findSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106575:26)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:85
    at getSourceFileFromReferenceWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106499:34)
    at processSourceFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106532:13)
    at processRootFile (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:106342:13)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:105207:60
    at Object.forEach (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:188:30)
    at Object.createProgram (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:105207:16)
    at synchronizeHostData (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:146570:26)
    at Proxy.getProgram (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:146662:13)
    at InferredProject.Project.updateGraphWorker (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:151524:53)
    at InferredProject.Project.updateGraph (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:151457:42)
    at updateProjectIfDirty (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:152922:45)
    at Array.forEach (<anonymous>)
    at ProjectService.assignProjectToOpenedScriptInfo (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:155099:41)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:155385:97
    at Object.flatMap (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:513:25)
    at ProjectService.applyChangesInOpenFiles (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:155385:41)
    at Session.handlers.ts.Map.ts.getEntries._a.<computed> (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:156403:46)
    at /snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:158276:88
    at IOSession.Session.executeWithRequestId (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:158267:28)
    at IOSession.Session.executeCommand (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:158276:33)
    at IOSession.Session.onMessage (/snap/code/52/usr/share/code/resources/app/extensions/node_modules/typescript/lib/tsserver.js:158300:35)
```

**Comment by kitsonk** [comment]
What version of Deno are you using?

What does your project setup look like (e.g. what file did you have open when the error occurred, or what steps can be taken to reproduce it?)

**Comment by ynwd** [comment]
> What version of Deno are you using?

1.6.3

> 
> What does your project setup look like (e.g. what file did you have open when the error occurred, or what steps can be taken to reproduce it?)

Just create simple hello-world-web deno project.
https://deno.land/manual/examples/hello_world

Add some functionality, and a few minutes later (about 5-15 minutes), the error appears.

**Comment by ynwd** [comment]
Additional info:

```
Starting Deno language server...
  version: 1.6.3 (release, x86_64-unknown-linux-gnu)
Connected to "vscode" 1.52.1
language server initialized
Server ready.
thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', cli/lsp/tsc.rs:771:43
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
[Info  - 2:58:34 PM] Connection to server got closed. Server will restart.
Starting Deno language server...
  version: 1.6.3 (release, x86_64-unknown-linux-gnu)
Connected to "vscode" 1.52.1
language server initialized
Server ready.
thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', cli/lsp/tsc.rs:771:43
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
[Info  - 2:58:43 PM] Connection to server got closed. Server will restart.
[Error - 2:58:43 PM] Request textDocument/hover failed.
Error: Connection got disposed.
	at Object.dispose (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/main.js:904:25)
	at Object.dispose (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-languageclient/lib/client.js:74:35)
	at LanguageClient.handleConnectionClosed (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-languageclient/lib/client.js:2309:42)
	at LanguageClient.handleConnectionClosed (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-languageclient/lib/main.js:155:15)
	at closeHandler (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-languageclient/lib/client.js:2296:18)
	at CallbackList.invoke (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/events.js:62:39)
	at Emitter.fire (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/events.js:121:36)
	at closeHandler (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/main.js:240:26)
	at CallbackList.invoke (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/events.js:62:39)
	at Emitter.fire (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/events.js:121:36)
	at StreamMessageReader.fireClose (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/messageReader.js:111:27)
	at Socket.<anonymous> (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/messageReader.js:151:46)
	at Socket.emit (events.js:228:7)
	at Pipe.<anonymous> (net.js:664:12)
Starting Deno language server...
  version: 1.6.3 (release, x86_64-unknown-linux-gnu)
Connected to "vscode" 1.52.1
language server initialized
Server ready.
thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', cli/lsp/tsc.rs:771:43
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
fatal runtime error: failed to initiate panic, error 5
[Info  - 3:00:01 PM] Connection to server got closed. Server will restart.
[Error - 3:00:01 PM] Request textDocument/documentHighlight failed.
Error: Connection got disposed.
	at Object.dispose (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/main.js:904:25)
	at Object.dispose (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-languageclient/lib/client.js:74:35)
	at LanguageClient.handleConnectionClosed (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-languageclient/lib/client.js:2309:42)
	at LanguageClient.handleConnectionClosed (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-languageclient/lib/main.js:155:15)
	at closeHandler (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-languageclient/lib/client.js:2296:18)
	at CallbackList.invoke (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/events.js:62:39)
	at Emitter.fire (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/events.js:121:36)
	at closeHandler (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/main.js:240:26)
	at CallbackList.invoke (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/events.js:62:39)
	at Emitter.fire (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/events.js:121:36)
	at StreamMessageReader.fireClose (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/messageReader.js:111:27)
	at Socket.<anonymous> (/home/e560/.vscode/extensions/denoland.vscode-deno-canary-0.0.6/client/node_modules/vscode-jsonrpc/lib/messageReader.js:151:46)
	at Socket.emit (events.js:228:7)
	at Pipe.<anonymous> (net.js:664:12)
Starting Deno language server...
  version: 1.6.3 (release, x86_64-unknown-linux-gnu)
Connected to "vscode" 1.52.1
language server initialized
Server ready.
```

**Comment by kitsonk** [comment]
Ok, do you get that at the same time you get the other error, with it just sitting there 5-10 after you start?  The `thread '<unnamed>' panicked at 'called "Option::unwrap()" on a "None" value', cli/lsp/tsc.rs:771:43` is a known bug (denoland/deno#8881) which I am working on a fix for.  It is an internal race condition.  It is possible that that then is causing some sort of knock on effect.

What does your workspace settings look like `/.vscode/settings.json`?

**Comment by ynwd** [comment]
> Ok, do you get that at the same time you get the other error, with it just sitting there 5-10 after you start? The `thread '<unnamed>' panicked at 'called "Option::unwrap()" on a "None" value', cli/lsp/tsc.rs:771:43` is a known bug ([denoland/deno#8881](https://github.com/denoland/deno/issues/8881)) which I am working on a fix for. It is an internal race condition. It is possible that that then is causing some sort of knock on effect.
> 
> What does your workspace settings look like `/.vscode/settings.json`?

```
{
  "deno.enable": true,
  "deno.lint": true,
  "deno.unstable": true,
  "deno.import_intellisense_origins": {
    "https://deno.land": true,
  },
  "[typescript]": {
    "editor.defaultFormatter": "denoland.vscode-deno",
  },
  "[typescriptreact]": {
    "editor.defaultFormatter": "denoland.vscode-deno",
  },
}
```

**Comment by kitsonk** [comment]
Ok, thank you... Nothing unexpected there...

**Comment by danopia** [comment]
Is this related? Similar message but the stack looks different.

```
ERROR TSLS = Error: Debug Failure. False expression.
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error: Debug Failure. False expression.
    at checkChangeRange (deno:cli/tsc/00_typescript.js:35398:26)
    at Object.updateSourceFile (deno:cli/tsc/00_typescript.js:34989:13)
    at Object.updateSourceFile (deno:cli/tsc/00_typescript.js:28469:47)
    at Object.updateLanguageServiceSourceFile (deno:cli/tsc/00_typescript.js:146911:40)
    at acquireOrUpdateDocument (deno:cli/tsc/00_typescript.js:121308:43)
    at Object.updateDocumentWithKey (deno:cli/tsc/00_typescript.js:121274:20)
    at Object.getOrCreateSourceFileByPath [as getSourceFileByPath] (deno:cli/tsc/00_typescript.js:147219:49)
    at tryReuseStructureFromOldProgram (deno:cli/tsc/00_typescript.js:105837:28)
    at Object.createProgram (deno:cli/tsc/00_typescript.js:105363:29)
    at synchronizeHostData (deno:cli/tsc/00_typescript.js:147141:26)', cli/lsp/language_server.rs:800:66
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by kitsonk** [comment]
@danopia likely, did you have any multi-byte characters/emoji, even in strings, in the document you were editing?

**Comment by danopia** [comment]
I don't remember the exact file at the time, but it's pretty likely; I'm porting some old javascript into TS and there's emojis in most of the files. So I've been seeing the known LSP edit alignment issues from that.

**Comment by kitsonk** [comment]
@danopia yeah, I am hopefully going to fix it this week.  I know it is sort of painful right now.

**Comment by kitsonk** [comment]
I think these issues are resolved in 3.0.1 of the extension and Deno 1.7.5. If things like this happen again, let's open a new issue.
